pyproject.toml

```diff
@@ -20,7 +20,7 @@ dependencies = [
     "confluent-kafka==2.3.0",
     "opentelemetry-api==1.20.0",
     "opentelemetry-sdk==1.20.0",
-    "pydantic==2.5.2",
+    "telicent-label-builder==0.1.0"
```

Comment:
Where doe this dependency actually get used? Is it only in tests?

Reply:
Yes it is only test, perhaps it should be moved to extra dev.